style/js/shield_defs.js

```diff
@@ -718,6 +718,51 @@ export function loadShields(shieldImages) {
   shields["US:SC:Business"] = banneredShield(shields["US:SC"], ["BUS"]);
   shields["US:SC:Alternate"] = banneredShield(shields["US:SC"], ["ALT"]);
 
+  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX:Loop"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["LOOP"]
+  );
```

Comment:
Texas has at least one city route, Andrews Loop 1910. [Here's what the shield looks like.](https://www.andrewscountynews.com/sites/andrewscountynews.etypegoogle12.com/files/4100009ffd_Ar00103004.jpg)

For `network=US:TX:Andrews:Andrews:Loop`, I think a shield matching the State Loop shield with dark blue instead of black would work okay. Obviously we can't include the copyrighted artwork, not that it's particularly helpful for navigation.